libs/protos/protos/steps/sp_steps_detective.proto

```diff
@@ -121,6 +124,11 @@ message DetectiveStepResultMatch {
 
   // Value of the match
   bytes value = 5;
+
+  // Used to return the type of PII we detected.
+  // This differs from `type` in that it is the type of PII we detected, not the type of match we used to detect it.
+  // This value is freeform and determined based on the values inside the keyword matcher.
+  string pii_type = 6;
```

Comment:
Should this be `map[string]string` (where key == path and value == entity that was found) since it's possible that multiple fields will be found?

Reply:
We return one DetectiveStepResultMatch for each match, so don't need to support multiple inside of DetectiveStepResultMatch